### ISSUE TITLE
This package [ logging-log4j-jmx-gui ] is not in the mvn repo?

### ISSUE THREAD

**Issue by MoonLord-LM** [apache/logging-log4j2#1926]
HTTP 404：
https://repo.maven.apache.org/maven2/org/apache/logging/log4j/log4j-jmx-gui/2.21.1/

**Comment by MoonLord-LM** [comment]
<img width="1173" alt="Screenshot 2023-10-31 025833" src="https://github.com/apache/logging-log4j2/assets/8104133/096c81e1-8849-4cd2-b1e1-23b230b720e5">

**Comment by MoonLord-LM** [comment]
https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-bom/2.21.1

**Comment by vy** [comment]
Since Log4j `2.21.0` release, Log4j JMX GUI is not a part of the official Log4j distribution. (See [release notes](https://logging.apache.org/log4j/2.x/release-notes.html#release-notes-2-21-0).) It has been moved to [a separate repository](/apache/logging-log4j-jmx-gui) and started getting released there with its own life cycle.

Though there was a slight problem with the `2.21.0` release of Log4j JMX GUI: we got the `groupId` wrong and it ended up inheriting `org.apache.logging` from parent, whereas it should have been `org.apache.logging.log4j`.

In conclusion, Log4j JMX GUI...
1. has [a new home](/apache/logging-log4j-jmx-gui) and release life cycle
2. version `2.21.0` is available at `org.apache.logging:log4j-jmx-gui` coordinate
3. version `2.21.1` will move the coordinate back to `org.apache.logging.log4j:log4j-jmx-gui`
